### PR TITLE
BLADERUNNER: Fix detection ADGF flags

### PR DESCRIPTION
**Pull request by tsoliman**

### CODE REVIEW

**Comment by peterkohaut** [comment]
LGTM, you were faster than me :)

File: engines/bladerunner/detection_tables.h [hunk 1]
```diff
@@ -118,7 +118,7 @@ static const ADGameDescription gameDescriptions[] = {
 		AD_ENTRY1s("STARTUP.MIX", "5643b53306ca7764cf1ec7b79c9630a3", 2312374),
 		Common::EN_ANY,
 		Common::kPlatformWindows,
-		ADGF_NO_FLAGS,
+		ADGF_TESTING,
```
Comment (peterkohaut) [inline engines/bladerunner/detection_tables.h]: actually this final/enhanced version is not yet ready for testing

File: engines/bladerunner/detection_tables.h [hunk 2]
```diff
@@ -129,7 +129,7 @@ static const ADGameDescription gameDescriptions[] = {
 		AD_ENTRY1s("STARTUP.MIX", "57d674ed860148a530b7f4957cbe65ec", 2314301),
 		Common::DE_DEU,
 		Common::kPlatformWindows,
-		ADGF_NO_FLAGS,
+		ADGF_TESTING,
```
Comment (peterkohaut) [inline engines/bladerunner/detection_tables.h]: not yet ready

File: engines/bladerunner/detection_tables.h [hunk 3]
```diff
@@ -140,7 +140,7 @@ static const ADGameDescription gameDescriptions[] = {
 		AD_ENTRY1s("STARTUP.MIX", "39d1901df50935d58aee252707134952", 2314526),
 		Common::FR_FRA,
 		Common::kPlatformWindows,
-		ADGF_NO_FLAGS,
+		ADGF_TESTING,
```
Comment (peterkohaut) [inline engines/bladerunner/detection_tables.h]: not yet ready

File: engines/bladerunner/detection_tables.h [hunk 4]
```diff
@@ -151,7 +151,7 @@ static const ADGameDescription gameDescriptions[] = {
 		AD_ENTRY1s("STARTUP.MIX", "c7ceb9c691223d25e78516aa519ff504", 2314461),
 		Common::IT_ITA,
 		Common::kPlatformWindows,
-		ADGF_NO_FLAGS,
+		ADGF_TESTING,
```
Comment (peterkohaut) [inline engines/bladerunner/detection_tables.h]: not yet ready

File: engines/bladerunner/detection_tables.h [hunk 5]
```diff
@@ -162,7 +162,7 @@ static const ADGameDescription gameDescriptions[] = {
 		AD_ENTRY1s("STARTUP.MIX", "c198b54a5366b88b1734bbca21d3b192", 2678672),
 		Common::RU_RUS,
 		Common::kPlatformWindows,
-		ADGF_NO_FLAGS,
+		ADGF_TESTING,
```
Comment (peterkohaut) [inline engines/bladerunner/detection_tables.h]: not yet ready

File: engines/bladerunner/detection_tables.h [hunk 6]
```diff
@@ -173,7 +173,7 @@ static const ADGameDescription gameDescriptions[] = {
 		AD_ENTRY1s("STARTUP.MIX", "bf42af841d9f4b643665013a348c81e0", 2483111),
 		Common::RU_RUS,
 		Common::kPlatformWindows,
-		ADGF_NO_FLAGS,
+		ADGF_TESTING,
 		GUIO2(GAMEOPTION_SITCOM, GAMEOPTION_SHORTY)
```
Comment (peterkohaut) [inline engines/bladerunner/detection_tables.h]: not yet ready

**Comment by peterkohaut** [comment]
any update?

**Comment by tsoliman** [comment]
Yeah I'll fix it when I go home. I am in the US. Chicago time. GMT +5/6

File: engines/bladerunner/detection_tables.h (hunk 1 above)
Comment (criezy) [reply engines/bladerunner/detection_tables.h]: You might instead want to add the missing ADGF_UNSTABLE flag for this one and the other entries not yet ready for testing.

**Comment by tsoliman** [comment]
changed the `bladerunner-final` entries to `ADGF_UNSTABLE`

**Pull request merged**